ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -122,6 +122,17 @@ router bgp {{ router_bgp.as }}
 {%         if router_bgp.neighbors[neighbor].route_map_in is arista.avd.defined %}
    neighbor {{ neighbor }} route-map {{ router_bgp.neighbors[neighbor].route_map_in }} in
 {%         endif %}
+{%         if router_bgp.neighbors[neighbor].default_originate is arista.avd.defined %}
+{%             set neighbor_default_originate_router_bgp = "neighbor " ~ peer_group ~ " default-originate" %}
```

Comment:
Just a suggestion, you can keep this var name simple and end with ```_cli``` like ```ne_default_originate_cli``` or something which you think looks good, which gives an idea that this is for cli generation rather than using it for some other evaluation. As this template is for router-bgp, so we can remove ```router_bgp``` in the var name as I don't see any essence of it. Overall, try to keep the var name as short as possible and easy to understand.

Reply:
IMO `default_originate_cli` would be good.